SIL.WritingSystems/LdmlDataMapper.cs

```diff
@@ -819,6 +827,7 @@ public void Write(string filePath, WritingSystemDefinition ws, Stream oldFile)
 				using (var writer = XmlWriter.Create(filePath, writerSettings))
 				{
 					WriteLdml(writer, element, ws);
+					writer.Flush();
```

Comment:
The errors I've seen have been caused by the write being done, but the disk file not being written to - at least in Windows the write to disk is normally deferred for an unspecified time. From my reading of the Flush on the Writer, it seems to do the force the disk update.

I created a FlushingStream for Paratext and this has prevented many errors related to files just containing nulls.